packages/storybook-react/src/stories/ButtonGroup.stories.tsx

```diff
@@ -27,7 +27,7 @@ const meta = {
     docs: {
       page: () => (
         <>
-          <Description markdown={readme} />
+          <Description>{readme}</Description>
           <Primary />
           <ArgsTable story={PRIMARY_STORY} />
           <Stories />
```

Comment:
ButtonGroup geeft bij mij een foutmelding bij de rendering van parameters.docs.page: e.background is undefined